### PR TITLE
Add a contributing file, some TOCs and a section for the showcase

### LINKED ISSUE #14: [BUG] Contributing guidelines
Need contributing guidelines, whether for code, testing, feature requests, suggestions, pointers to usage by other projects, etc.

### PR DESCRIPTION
**Pull request by lornajane**
Add a CONTRIBUTING file to show how to use the project locally, encourage bugs/issues and patches, and point people to the showcase.

Fixes #14

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Contributing to Go-Streamdeck
+
+All issues, pull requests and other feedback are very welcome in this project. Please do take the time to read the information below, and get involved!
+
+- [Use Go-Streamdeck From Source](#use-go-streamdeck-from-source)
+- [Share Issues, Bugs and Feature Requests](#share-issues-bugs-and-feature-requests)
+- [Submit a Patch or Pull Request](#submit-a-patch-or-pull-request)
+- [Try the Examples](#try-the-examples)
+- [Usage in Other Projects](#usage-in-other-projects)
+-
```
Comment (magicmonkey) [inline CONTRIBUTING.md]: Erroneous bullet point, to be removed

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+## Try the Examples
+
+The `examples/` directory has a series of examples that may be useful in your own applications. Try these out to see the various features in action.
+
+Please open an issue to update us for which operating systems and StreamDeck combinations are working well for you!
+
+## Usage in Other Projects
+
+You are very welcome to use this project within the terms of the license. We would love to have you in our showcase, please open a pull request to update the `REAMDME.md` file with a link to your project.
```
Comment (magicmonkey) [inline CONTRIBUTING.md]: Typo on the word "README"

**Review by magicmonkey — CHANGES_REQUESTED** [review]
Looks good, thanks!  Couple of typos to fix, otherwise is great.

File: CONTRIBUTING.md (hunk 2 above)
Comment (lornajane) [reply CONTRIBUTING.md]: fixed!

File: CONTRIBUTING.md (hunk 1 above)
Comment (lornajane) [reply CONTRIBUTING.md]: removed!

**Pull request merged**